package.json

```diff
@@ -12,7 +12,8 @@
     "contributors": "githubcontrib --owner angular --repo angular-ja --cols 6 --format md --showlogin true --sortOrder desc | pbcopy",
     "update-origin": "./update-origin.sh",
     "textlint": "textlint 'aio-ja/**/!(*.en.*)'",
-    "textlint:f": "textlint --fix 'aio-ja/**/!(*.en.*)'"
+    "textlint:f": "textlint --fix 'aio-ja/**/!(*.en.*)'",
+    "postinstall": "opencollective postinstall"
```

Comment:
This repository is not for publish library. The people who execute "install" are all contributors. I think this suggestion is unneeded.

Reply:
Instead, this repository is a source of https://angular.jp website.
So I would like put "donate button" into the top page or somewhere.
is there any guide to do that?